composer.json

```diff
@@ -13,16 +13,13 @@
         }
     ],
     "require": {
-        "matthiasnoback/symfony-config-test": "0.*|~1.0",
-        "symfony/dependency-injection": "^2.0.5|~3.0",
-        "symfony/config": "^2.0.5|~3.0",
-        "sebastian/exporter": "~1"
+        "matthiasnoback/symfony-config-test": "^1.0|^2.0",
+        "phpunit/phpunit": "^4.0|^5.0",
```

Comment:
This has nothing to do here and was especially removed on #47.

@matthiasnoback Could you please revert this back?